<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>371</width>
    <height>152</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {background-color:rgb(191, 199, 200)}</string>
  </property>
  <property name="locale">
   <locale language="German" country="Germany"/>
  </property>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>351</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QProgressBar {
                          background-color: #74c8ff;
                          color: #0a9dff;
                          border-style: outset;
                          border-width: 2px;
                          border-color: #74c8ff;
                          border-radius: 7px;
                          text-align: left; }

QProgressBar::chunk {
                          background-color: #010327; }</string>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Info:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>100</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {border:3px; border-radius:12px;background-color:#193d5c;color:white}
QPushButton:hover{background-color:#3498DB}
</string>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {border:3px; border-radius:12px;background-color:#193d5c;color:white}
QPushButton:hover{background-color:#3498DB}
</string>
   </property>
   <property name="text">
    <string>Suchen</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>100</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {border:3px; border-radius:12px;background-color:#193d5c;color:white}
QPushButton:hover{background-color:#3498DB}
</string>
   </property>
   <property name="text">
    <string>Beenden</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>CSV-Path:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>10</y>
     <width>271</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {background-color:rgb(34, 56, 135);border-radius:10px;border: 1px solid black;color:white}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
